Bigtable/src/Connection/Grpc.php

```diff
@@ -251,15 +251,37 @@ public function deleteAppProfile(array $args)
      */
     public function getIamPolicy(array $args)
     {
-        throw new \BadMethodCallException('This method is not implemented yet');
+        $resource = $this->pluck('resource', $args);
```

Comment:
This is already checked in by me. Same for setIamPolicy and others.
IMHO, you should just focus on your current methods in this PR. Or else it will never get merged.